nifi-nar-bundles/nifi-registry-bundle/nifi-registry-service/pom.xml

```diff
@@ -52,5 +57,22 @@ language governing permissions and limitations under the License. -->
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.networknt</groupId>
+            <artifactId>json-schema-validator</artifactId>
+            <version>1.0.86</version>
+        </dependency>
     </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>8</source>
+                    <target>8</target>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
```

Comment:
This configuration should not be necessary since the minimum Java version is 21 in the root Maven configuration.

Reply:
I have removed that